identity_test.go

```diff
@@ -0,0 +1,30 @@
+package e3db
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestToznySDKV3_Login(t *testing.T) {
+	request := TozIDLoginRequest{
+		Username:     "",
+		Password:     "",
+		RealmName:    "",
+		APIBaseURL:   "https://api.e3db.com",
+		LoginHandler: mfaHandler,
+	}
+	sdk, err := GetSDKV3ForTozIDUser(request)
+	if err != nil {
+		t.Fatal("Abort", err)
+	}
+	fmt.Printf("%v", sdk)
+}
```

Comment:
does this test work? It looks to me like a helper test for use during development but not something to merge in.

Reply:
This should get removed, and will be default when the conflicts get resolved